frontend/src/AboutPage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export class AboutPage extends React.Component<any, any>{
```

Comment:
Do we want to write it verbose or omit default stuff?
```suggestion
export class AboutPage extends React.Component{
```

Reply:
done